backend/src/repositories/base/base.repository.js

```diff
@@ -47,20 +47,18 @@ class BaseRepository {
     logger.info(
       `==================== ${this.constructor.name}, call method Update ====================`
     );
-    let result = await this.repos.update(entity, {
+    await this.repos.update(entity, {
```

Comment:
does it return updated entity? I think it is not good when returning param as result